Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBServiceTests/SyncPerformanceIntTestHarness.swift

```diff
@@ -0,0 +1,363 @@
+// swiftlint:disable force_try
+import XCTest
+import MongoSwift
+import StitchCore
+import StitchCoreSDK
+import StitchCoreAdminClient
+import StitchDarwinCoreTestUtils
+@testable import StitchCoreRemoteMongoDBService
+import StitchCoreLocalMongoDBService
+@testable import StitchRemoteMongoDBService
+
+typealias TestDefinition = (_ ctx: SyncPerformanceContext, _ numDoc: Int, _ docSize: Int) throws -> Void
+typealias SetupDefinition = (_ ctx: SyncPerformanceContext, _ numDoc: Int, _ docSize: Int) throws -> Void
+typealias TeardownDefinition = (_ ctx: SyncPerformanceContext, _ numDoc: Int, _ docSize: Int) throws -> Void
+
+// This is how we want to do things once we create the new test scheme and import the .h file as an
+// objective-c bridging header
+// let testStitchAPIKey = PERF_IOS_API_KEY.isEmpty ?
+//    ProcessInfo.processInfo.environment["PERF_IOS_API_KEY"] : PERF_IOS_API_KEY
+
+class SyncPerformanceIntTestHarness: BaseStitchIntTestCocoaTouch {
```

Comment:
[q] is it still necessary to inherit this class?

Reply:
Here it sadly is